src/shared/components/challenge-detail/Submissions/index.jsx

```diff
@@ -19,7 +19,7 @@ function renderSubmission(s) {
       <div styleName="bottom-info">
         <div styleName="links">
           <a target="_blank" href={`${config.URL.STUDIO}?module=DownloadSubmission&sbmid=${s.submissionId}`}>{`#${s.submissionId}`}</a>
-          <a styleName="handle" target="_blank" href={`${config.URL.BASE}/member-profile/${s.submitter}/design`}>{s.submitter}</a>
+          <a styleName="handle" target="_blank" href={`${config.URL.BASE}/member-profile/${s.submitter}`}>{s.submitter}</a>
```

Comment:
I believe, should be `${config.URL.BASE}/members/${s.submitter}`, as it is where we got redirected anyway, if do the actions from the issue description. So, we can just go directly there. Please update